### PR TITLE
Revamp UI admin

### PR DESCRIPTION
**Pull request by revengemiroz**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

## Showcase

Please attach a picture or a video showing what this change does.

### CODE REVIEW

**Comment by ivanms1** [comment]
Task linked: [CU-2jn7116 Handle like and follow for not logged in user](https://app.clickup.com/t/2jn7116)

File: apps/admin/pages/projects/index.tsx [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+    return addApolloState(client, {
+      props: {
+        projects: data?.data?.getProjectsAdmin?.results,
+      },
+      revalidate: 60,
+    });
+  } catch (error) {
+    return {
+      props: {
+        data: error.message,
```
Comment (ivanms1) [inline apps/admin/pages/projects/index.tsx]: why are you passing this?
you can remove the ssr here, not necessary for the admin page

File: apps/admin/pages/users/index.tsx [hunk 2]
```diff
@@ -1,3 +1,31 @@
+import { addApolloState, initializeApollo } from 'apollo';
 import Users from '@/pages/Users';
 
 export default Users;
+
+import QUERY_GET_USERS_ADMIN from './queryGetUsers.graphql';
+
+export async function getStaticProps() {
```
Comment (ivanms1) [inline apps/admin/pages/users/index.tsx]: remove, it's ok for the admin page to be client side

File: apps/admin/src/components/SideNav/SideNav.tsx [hunk 3]
```diff
@@ -7,6 +7,7 @@ import Button from '../../components/Button';
 import Settings from '../../../public/assets/settings.svg';
 import LogOut from '../../../public/assets/log-out.svg';
 import Dashboard from '../../../public/assets/dashboard.svg';
+import Users from '../../../public/assets/users.svg';
```
Comment (ivanms1) [inline apps/admin/src/components/SideNav/SideNav.tsx]: setup absolute imports
https://nextjs.org/docs/advanced-features/module-path-aliases

File: apps/admin/src/components/Table/Pagination.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+function Pagination({ table }) {
+  console.log(
```
Comment (ivanms1) [inline apps/admin/src/components/Table/Pagination.tsx]: remove please

File: apps/admin/src/components/Table/Table.tsx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+    getFilteredRowModel: getFilteredRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+    getPaginationRowModel: getPaginationRowModel(),
+    getCoreRowModel: getCoreRowModel(),
+    enableColumnFilters: true,
+    columnResizeMode: 'onChange',
+  });
+
+  useEffect(() => {
+    // table.setState(5);
```
Comment (ivanms1) [inline apps/admin/src/components/Table/Table.tsx]: what is this empty useEffect?

File: apps/admin/src/components/Table/Table.tsx [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+            <thead className=''>
+              {table.getHeaderGroups().map((headerGroup) => (
+                <tr key={headerGroup.id}>
+                  {headerGroup.headers.map((header) => (
+                    <th
+                      className='border-b border-gray-200 pr-14 pb-[10px] text-start sticky'
+                      key={header.id}
+                      colSpan={header.colSpan}
+                      style={{
+                        width: header.getSize() ? header.getSize() : undefined,
```
Comment (ivanms1) [inline apps/admin/src/components/Table/Table.tsx]:
```suggestion
                        width: header.getSize() || undefined
```

File: apps/admin/src/pages/Projects/index.tsx [hunk 7]
```diff
@@ -1,10 +1,156 @@
...
+  projectName: string;
+  preview: string;
+  description: string;
+  submissionDate: number;
+  author: string;
+  authorEmail: string;
+  authorAvatar: string;
+};
+
+const columns = [
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/index.tsx]: memoize this

File: apps/admin/src/pages/Projects/index.tsx [hunk 8]
```diff
@@ -1,10 +1,156 @@
...
+  {
+    accessorKey: 'author',
+    width: 200,
+    header: () => <span>Author</span>,
+    cell: (info) => {
+      return (
+        <div className='flex flex-row gap-[10px] items-center'>
+          <div className='flex max-h-[70px] max-w-[70px]'>
+            <Image
+              loader={() => info?.row?.original?.author?.avatar}
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/index.tsx]: is this needed?

File: apps/admin/src/pages/Projects/index.tsx [hunk 9]
```diff
@@ -1,10 +1,156 @@
...
+              src={info?.row?.original?.author?.avatar}
+              alt={info?.row?.original?.author?.avatar}
+              width={'100%'}
+              height={'100%'}
+              style={{
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/index.tsx]: no inline styles

File: apps/admin/src/pages/Projects/index.tsx [hunk 10]
```diff
@@ -1,10 +1,156 @@
...
+  },
+  {
+    accessorKey: 'createdAt',
+    header: () => <span>Submission Date</span>,
+    size: 10,
+    maxSize: 10,
+    cell: (info) => {
+      return (
+        <span className='text-gray-700 font-bold text-[14px]'>
+          {new Date(info?.getValue()).toDateString()}
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/index.tsx]: format to something more readable

File: apps/admin/src/pages/Projects/index.tsx [hunk 11]
```diff
@@ -1,10 +1,156 @@
...
+    header: () => <span>Action</span>,
+    maxWidth: '100px',
+    cell: (info) => {
+      return (
+        <div className='flex flex-row items-center gap-[20px] '>
+          <button
+            className={classNames(
+              'text-[14px] text-white font-bold py-[5px] px-[20px] rounded-full',
+              {
+                'bg-red-600': true,
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/index.tsx]: if this is always true you can just pass directly

File: apps/admin/src/pages/Projects/index.tsx [hunk 12]
```diff
@@ -1,10 +1,156 @@
...
+              }
+            )}
+            onClick={() => alert('delete button clicked')}
+          >
+            Delete
+          </button>
+          <Link href={info?.row?.original?.repoLink} passHref>
+            <a target='_blank' rel='noopener noreferrer'>
+              <i className='w-[20px] h-[20px] flex justify-center items-center '>
+                <GithubIcon className='fill-[#a3aed0]' />
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/index.tsx]: add color to tailwind config

File: apps/admin/src/pages/Users/index.tsx [hunk 13]
```diff
@@ -1,10 +1,184 @@
...
+  {
+    accessorKey: 'action',
+    header: () => <span>Action</span>,
+    cell: (info) => {
+      return (
+        <button
+          className={classNames(
+            'text-[14px] text-white font-bold py-[5px] px-[20px] rounded-full',
+            {
+              'bg-red-600': true,
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/index.tsx]: same as above, just pass it on the string

File: apps/admin/src/pages/Users/index.tsx [hunk 14]
```diff
@@ -1,10 +1,184 @@
...
+          onClick={() => alert('The user has been banned')}
+        >
+          Ban
+        </button>
+      );
+    },
+  },
+];
+
+function Index({ users }, props) {
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/index.tsx]:
```suggestion
function Users({ users }, props) {
```

**Review by ivanms1 — CHANGES_REQUESTED** [review]

File: apps/admin/src/components/SideNav/SideNav.tsx (hunk 3 above)
Comment (revengemiroz) [reply apps/admin/src/components/SideNav/SideNav.tsx]: done

File: apps/admin/src/components/Table/Table.tsx (hunk 5 above)
Comment (revengemiroz) [reply apps/admin/src/components/Table/Table.tsx]: removed

File: apps/admin/src/components/Table/Table.tsx (hunk 6 above)
Comment (revengemiroz) [reply apps/admin/src/components/Table/Table.tsx]: done

File: apps/admin/src/pages/Projects/index.tsx (hunk 7 above)
Comment (revengemiroz) [reply apps/admin/src/pages/Projects/index.tsx]: done

File: apps/admin/src/pages/Projects/index.tsx (hunk 11 above)
Comment (revengemiroz) [reply apps/admin/src/pages/Projects/index.tsx]: done

File: apps/admin/src/pages/Users/index.tsx (hunk 13 above)
Comment (revengemiroz) [reply apps/admin/src/pages/Users/index.tsx]: done

File: apps/admin/src/pages/Users/index.tsx (hunk 14 above)
Comment (revengemiroz) [reply apps/admin/src/pages/Users/index.tsx]: done

File: apps/admin/pages/users/index.tsx (hunk 2 above)
Comment (revengemiroz) [reply apps/admin/pages/users/index.tsx]: done

File: apps/admin/src/components/Table/Table.tsx [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+        <DebouncedInput
+          value={globalFilter ?? ''}
+          onChange={(value) => setGlobalFilter(String(value))}
+          placeholder='Search all columns'
+        />
+      </div>
+      <div className='h-full w-full max-w-full overflow-auto'>
+        {table.getRowModel().rows.length !== 0 && (
+          <table className='w-full relative'>
+            <thead className=''>
```
Comment (ivanms1) [inline apps/admin/src/components/Table/Table.tsx]: remove empty classname

File: apps/admin/src/components/Table/Table.tsx [hunk 16]
```diff
@@ -0,0 +1,124 @@
...
+
+    // Store the itemRank info
+    addMeta({
+      itemRank,
+    });
+
+    // Return if the item should be filtered in/out
+    return itemRank.passed;
+  };
+
```
Comment (ivanms1) [inline apps/admin/src/components/Table/Table.tsx]: what is this doing?

File: apps/admin/src/pages/Dashboard/Dashboard.tsx [hunk 17]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { NextSeo } from 'next-seo';
+
+function Index() {
```
Comment (ivanms1) [inline apps/admin/src/pages/Dashboard/Dashboard.tsx]:
```suggestion
function Dashboard() {
```

File: apps/admin/src/pages/Projects/Projects.tsx [hunk 18]
```diff
@@ -0,0 +1,205 @@
...
+import GithubIcon from '@/public/assets/github.svg';
+import ExternalLink from '@/public/assets/external-link.svg';
+import { Button, Modal, LoaderOverlay } from 'ui';
+
+import {
+  useDeleteProjectsMutation,
+  useGetProjectsAdminQuery,
+} from 'apollo-hooks';
+
+function Index() {
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/Projects.tsx]:
```suggestion
function Projects() {
```

File: apps/admin/src/pages/Projects/Projects.tsx [hunk 19]
```diff
@@ -0,0 +1,205 @@
...
+      {
+        accessorKey: 'title',
+        header: () => <span>Title</span>,
+        size: 300,
+        cell: (info) => {
+          return (
+            <div className='flex flex-row gap-[10px] items-center'>
+              <div className='flex max-h-[70px] max-w-[70px]'>
+                <Image
+                  loader={() => info?.row?.original?.preview}
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/Projects.tsx]: is this necessary?

File: apps/admin/src/pages/Projects/Projects.tsx [hunk 20]
```diff
@@ -0,0 +1,205 @@
...
+                  src={info?.row?.original?.preview}
+                  alt={info?.row?.original?.preview}
+                  width={100}
+                  height={100}
+                  style={{
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/Projects.tsx]: no inline styles

File: apps/admin/src/pages/Projects/Projects.tsx [hunk 21]
```diff
@@ -0,0 +1,205 @@
...
+      {
+        accessorKey: 'author',
+        width: 200,
+        header: () => <span>Author</span>,
+        cell: (info) => {
+          return (
+            <div className='flex flex-row gap-[10px] items-center'>
+              <div className='flex max-h-[70px] max-w-[70px]'>
+                <Image
+                  loader={() => info?.row?.original?.author?.avatar}
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/Projects.tsx]: remove if not necessary

File: apps/admin/src/pages/Projects/Projects.tsx [hunk 22]
```diff
@@ -0,0 +1,205 @@
...
+                  src={info?.row?.original?.author?.avatar}
+                  alt={info?.row?.original?.author?.avatar}
+                  width={'100%'}
+                  height={'100%'}
+                  style={{
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/Projects.tsx]: no inline styles

File: apps/admin/src/pages/Projects/Projects.tsx [hunk 23]
```diff
@@ -0,0 +1,205 @@
...
+        cell: (info) => {
+          return (
+            <div className='flex flex-row gap-[10px] items-center'>
+              <div className='flex max-h-[70px] max-w-[70px]'>
+                <Image
+                  loader={() => info?.row?.original?.author?.avatar}
+                  src={info?.row?.original?.author?.avatar}
+                  alt={info?.row?.original?.author?.avatar}
+                  width={'100%'}
+                  height={'100%'}
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/Projects.tsx]:
```suggestion
                  width="100%"
                  height="100%"
```
are you sure this works? I think you need to another prop to have dynamic width and heigth

File: apps/admin/src/pages/Projects/Projects.tsx [hunk 24]
```diff
@@ -0,0 +1,205 @@
+import React, { useState, useMemo } from 'react';
+import { toast } from 'react-hot-toast';
+import Image from 'next/image';
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/Projects.tsx]: use future image

File: apps/admin/src/pages/Projects/Projects.tsx [hunk 25]
```diff
@@ -0,0 +1,205 @@
...
+import { NextSeo } from 'next-seo';
+import Link from 'next/link';
+
+import Table from '../../components/Table';
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/Projects.tsx]: use the absolute imports

File: apps/admin/src/pages/Projects/Projects.tsx [hunk 26]
```diff
@@ -0,0 +1,205 @@
...
+
+import GithubIcon from '@/public/assets/github.svg';
+import ExternalLink from '@/public/assets/external-link.svg';
+import { Button, Modal, LoaderOverlay } from 'ui';
+
+import {
+  useDeleteProjectsMutation,
+  useGetProjectsAdminQuery,
+} from 'apollo-hooks';
+
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/Projects.tsx:20]: organize imports please

File: apps/admin/src/pages/Projects/Projects.tsx [hunk 27]
```diff
@@ -0,0 +1,205 @@
...
+                    onClick={() => deleteProjectClick(info?.row?.original?.id)}
+                  >
+                    Yes
+                  </Button>
+                  <Button onClick={() => setOpenDeleteModal(false)}>No</Button>
+                </div>
+              </Modal>
+
+              <button
+                className={classNames(
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/Projects.tsx]: classNames is not necessary here

File: apps/admin/src/pages/Users/Users.tsx [hunk 28]
```diff
@@ -0,0 +1,256 @@
...
+      maxWidth: '120px',
+      cursor: isDisabled ? 'not-allowed' : 'pointer',
+      background: isDisabled ? '#fff' : 'pointer',
+    };
+  },
+};
+
+function Users() {
+  const [updateUserAsAdmin] = useUpdateUserAsAdminMutation({
+    refetchQueries: 'active',
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/Users.tsx]: why?

File: apps/admin/src/pages/Users/Users.tsx [hunk 29]
```diff
@@ -0,0 +1,256 @@
...
+  });
+
+  const { data, loading } = useGetUserQuery();
+
+  const updateUser = async (user, role, action) => {
+    try {
+      if (action == 'BAN') {
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/Users.tsx]: use enums

File: apps/admin/src/pages/Users/Users.tsx [hunk 30]
```diff
@@ -0,0 +1,256 @@
...
+                <Image
+                  loader={() => info?.row?.original?.avatar}
+                  src={info?.row?.original?.avatar}
+                  alt={info?.row?.original?.avatar}
+                  width={'100%'}
+                  height={'100%'}
+                  style={{
+                    borderRadius: '50%',
+                  }}
+                />
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/Users.tsx]: same comments as image above

File: apps/admin/src/pages/Users/Users.tsx [hunk 31]
```diff
@@ -0,0 +1,256 @@
...
+                <span className='w-full text-gray-700 font-bold text-[17px]'>
+                  {info?.getValue()}
+                </span>
+                <div className='gap-[2px] flex flex-col'>
+                  <span className='w-full text-gray-700 font-medium text-[15px]'>
+                    {info?.row?.original?.email}
+                  </span>
+
+                  <Link
+                    href={`https://github.com/` + info?.row?.original?.github}
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/Users.tsx]: use template literals

File: apps/admin/src/pages/Users/Users.tsx [hunk 32]
```diff
@@ -0,0 +1,256 @@
...
+          );
+        },
+      },
+      {
+        accessorKey: 'createdAt',
+        header: () => <span>Created Date</span>,
+        cell: (info) => {
+          return (
+            <span className='text-gray-700 font-bold text-[14px]'>
+              {new Date(info?.getValue()).toDateString()}
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/Users.tsx]: format the dates
https://elijahmanor.com/blog/format-js-dates-and-times

File: apps/admin/src/pages/Users/Users.tsx [hunk 33]
```diff
@@ -0,0 +1,256 @@
...
+            value: info.getValue(),
+            label: info.getValue(),
+            isDisabled: true,
+          };
+
+          const options = [
+            {
+              value: 'USER',
+              label: 'USER',
+              isDisabled: info.getValue() == 'USER' ? true : false,
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/Users.tsx]:
```suggestion
              isDisabled: info.getValue() == 'USER',
```

Comment (ivanms1) [reply apps/admin/src/pages/Users/Users.tsx]: and use enum

File: apps/admin/src/pages/Users/Users.tsx [hunk 34]
```diff
@@ -0,0 +1,256 @@
...
+        accessorKey: 'action',
+        header: () => <span>Action</span>,
+        cell: (info) => {
+          return (
+            <button
+              className={classNames(
+                'text-[14px] text-white font-bold py-[5px] px-[20px] rounded-full',
+                {
+                  'bg-red-600': info?.row?.original?.banned == false,
+                  'bg-green-600': info?.row?.original?.banned == true,
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/Users.tsx]:
```suggestion
                  'bg-red-600': !info?.row?.original?.banned,
                  'bg-green-600': info?.row?.original?.banned,
```

File: packages/config/tailwind.config.js [hunk 35]
```diff
@@ -5,7 +5,6 @@ module.exports = {
       colors: {
         primary: '#A259FF',
         'primary-dark': '#9240FD',
```
Comment (ivanms1) [inline packages/config/tailwind.config.js:7]: add the new colors you use here

**Review by ivanms1 — CHANGES_REQUESTED** [review]

File: apps/admin/src/components/Table/Table.tsx (hunk 15 above)
Comment (revengemiroz) [reply apps/admin/src/components/Table/Table.tsx]: done

File: apps/admin/src/components/Table/Table.tsx (hunk 16 above)
Comment (revengemiroz) [reply apps/admin/src/components/Table/Table.tsx]: I dont know

Alt Text

File: apps/admin/next.config.js [hunk 36]
```diff
@@ -18,8 +18,11 @@ module.exports = withTM({
     domains: [
       'res.cloudinary.com',
       'avatars.githubusercontent.com',
+      'cloudflare-ipfs.com',
+      'loremflickr.com',
       'github.com',
       'github.githubassets.com',
+      'loremflickr.com',
```
Comment (ivanms1) [inline apps/admin/next.config.js]: this is added twice

File: apps/admin/src/components/BarGraph/index.tsx [hunk 37]
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {
+  AreaChart,
+  Area,
+  XAxis,
+  YAxis,
+  CartesianGrid,
```
Comment (ivanms1) [inline apps/admin/src/components/BarGraph/index.tsx]: remove if not used

File: apps/admin/src/components/BarGraph/index.tsx [hunk 38]
```diff
@@ -0,0 +1,85 @@
...
+  Tooltip,
+  Legend,
```
Comment (ivanms1) [inline apps/admin/src/components/BarGraph/index.tsx]: remove if not used

File: apps/admin/src/components/BarGraph/index.tsx [hunk 39]
```diff
@@ -0,0 +1,85 @@
...
+  ResponsiveContainer,
+} from 'recharts';
+
+function Index({ fillColor }) {
```
Comment (ivanms1) [inline apps/admin/src/components/BarGraph/index.tsx]:
```suggestion
function BarGraph({ fillColor }) {
```
also add interface

File: apps/admin/src/components/BarGraph/index.tsx [hunk 40]
```diff
@@ -0,0 +1,85 @@
...
+  console.log({ fillColor });
```
Comment (ivanms1) [inline apps/admin/src/components/BarGraph/index.tsx]: remove

File: apps/admin/src/components/BarGraph/index.tsx [hunk 41]
```diff
@@ -0,0 +1,85 @@
...
+  const data = [
```
Comment (ivanms1) [inline apps/admin/src/components/BarGraph/index.tsx]: move to constants and use uppecase for naming

File: apps/admin/src/components/BarGraph/index.tsx [hunk 42]
```diff
@@ -0,0 +1,85 @@
...
+      project: 3,
+    },
+    {
+      name: 'Saturday',
+      project: 7,
+    },
+  ];
+
+  return (
+    <div className='h-[100%] w-[100%]'>
```
Comment (ivanms1) [inline apps/admin/src/components/BarGraph/index.tsx]:
```suggestion
    <div className='h-full w-full'>
```

File: apps/admin/src/components/BarGraph/index.tsx [hunk 43]
```diff
@@ -0,0 +1,85 @@
...
+      <ResponsiveContainer width={'100%'} height={'100%'}>
```
Comment (ivanms1) [inline apps/admin/src/components/BarGraph/index.tsx]:
```suggestion
      <ResponsiveContainer width='100%' height='100%'>
```

File: apps/admin/src/components/BarGraph/index.tsx [hunk 44]
```diff
@@ -0,0 +1,85 @@
...
+                <stop offset='20%' stopColor={fillColor} stopOpacity={0.7} />
+                <stop offset='100%' stopColor='#FFFFFF' stopOpacity={0.6} />
+              </linearGradient>
+            </defs>
+          )}
+
+          <XAxis dataKey='name' />
+          <YAxis />
+          <Tooltip />
+          {/* <Legend /> */}
```
Comment (ivanms1) [inline apps/admin/src/components/BarGraph/index.tsx]: remove if not used

File: apps/admin/src/components/SideNav/SideNav.tsx [hunk 45]
```diff
@@ -64,7 +66,9 @@ const SideNav = () => {
           </Button>
         </Link>
         <Link href='/logout'>
-          <Button icon={<LogOut />}>Log Out</Button>
+          <Button icon={<LogOut />} onClick={() => signOut()}>
```
Comment (ivanms1) [inline apps/admin/src/components/SideNav/SideNav.tsx]:
```suggestion
          <Button icon={<LogOut />} onClick={signOut}>
```

File: apps/admin/src/components/Table/Pagination.tsx [hunk 46]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+function Pagination({ table }) {
```
Comment (ivanms1) [inline apps/admin/src/components/Table/Pagination.tsx]: this whole file is not used, remove please

File: apps/admin/src/pages/Dashboard/Dashboard.tsx (hunk 17 above)
Comment (ivanms1) [reply apps/admin/src/pages/Dashboard/Dashboard.tsx]: please change this

File: apps/admin/src/pages/Dashboard/Dashboard.tsx [hunk 47]
```diff
@@ -0,0 +1,86 @@
...
+  const { data } = useGetAllUsersQuery({
+    fetchPolicy: 'network-only',
+  });
+  const { data: projectsAdminData } = useGetProjectsAdminQuery();
+
+  console.log({ data });
+
+  return (
+    <div className='w-full h-full  flex flex-col border-2'>
+      {/* <p className='text-gray-900 text-3xl font-bold'>Dashboard</p> */}
```
Comment (ivanms1) [inline apps/admin/src/pages/Dashboard/Dashboard.tsx]: remove if not used

File: apps/admin/src/pages/Dashboard/Dashboard.tsx [hunk 48]
```diff
@@ -0,0 +1,86 @@
...
+import BarGraph from '../../components/BarGraph';
+import { useGetAllUsersQuery, useGetProjectsAdminQuery } from 'apollo-hooks';
+
+function Index() {
+  const { data } = useGetAllUsersQuery({
+    fetchPolicy: 'network-only',
+  });
+  const { data: projectsAdminData } = useGetProjectsAdminQuery();
+
+  console.log({ data });
```
Comment (ivanms1) [inline apps/admin/src/pages/Dashboard/Dashboard.tsx]: remove log

File: apps/admin/src/pages/Dashboard/Dashboard.tsx [hunk 49]
```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { NextSeo } from 'next-seo';
+
+import BarGraph from '../../components/BarGraph';
+import { useGetAllUsersQuery, useGetProjectsAdminQuery } from 'apollo-hooks';
+
+function Index() {
+  const { data } = useGetAllUsersQuery({
+    fetchPolicy: 'network-only',
```
Comment (ivanms1) [inline apps/admin/src/pages/Dashboard/Dashboard.tsx]: why?

File: apps/admin/src/pages/Dashboard/Dashboard.tsx [hunk 50]
```diff
@@ -0,0 +1,86 @@
...
+          <div className='flex flex-row text-[#242731] text-[20px] '>
+            <span className='font-bold'>
+              Project <span className='font-normal'>Statistics</span>
+            </span>
+          </div>
+          <BarGraph fillColor='#FF764C' />
+        </div>
+      </div>
+
+      {/* </div> */}
```
Comment (ivanms1) [inline apps/admin/src/pages/Dashboard/Dashboard.tsx]: remove

File: apps/admin/src/pages/Error/Error.tsx [hunk 51]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import { useRouter } from 'next/router';
+
+function Error(props) {
```
Comment (ivanms1) [inline apps/admin/src/pages/Error/Error.tsx]: remove props if not used

File: apps/admin/src/pages/Projects/Projects.tsx (hunk 26 above)
Comment (ivanms1) [reply apps/admin/src/pages/Projects/Projects.tsx:20]: address this please

File: apps/admin/src/pages/Projects/Projects.tsx [hunk 52]
```diff
@@ -0,0 +1,272 @@
...
+              </div>
+            </Modal>
+
+            <button
+              className={classNames(
+                'text-[14px] text-white font-bold py-[5px] px-[20px] rounded-full bg-red-600'
+              )}
+              onClick={() => {
+                deleteProjectClick(info?.row?.original?.id);
+                // setOpenDeleteModal(true);
```
Comment (ivanms1) [inline apps/admin/src/pages/Projects/Projects.tsx]: why is this commented?

File: apps/admin/src/pages/Users/Users.tsx [hunk 53]
```diff
@@ -0,0 +1,304 @@
...
+} from 'apollo-hooks';
+
+import Table from 'src/components/Table';
+
+import GithubIcon from '@/public/assets/github.svg';
+import useDebounce from 'src/components/Table/DebouncedInput';
+
+type Value = { value: string | number; label?: string };
+
+const options = [
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/Users.tsx]:
```suggestion
const OPTIONS = [
```

File: apps/admin/src/pages/Users/Users.tsx (hunk 29 above)
Comment (ivanms1) [reply apps/admin/src/pages/Users/Users.tsx]: address this

File: apps/admin/src/pages/Users/Users.tsx [hunk 54]
```diff
@@ -0,0 +1,304 @@
...
+          search,
+          order: sorting?.[0]?.desc ? SearchOrder.Desc : SearchOrder.Asc,
+          orderBy: sorting?.[0]?.id,
+          cursor: data?.getAllUsersAdmin?.nextCursor,
+        },
+      },
+    });
+  }, [debouncedSearchTerm, sorting]);
+
+  const updateUser = async (user, role, action) => {
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/Users.tsx]: make two separate functions, to update role and ban

File: apps/admin/src/pages/Users/Users.tsx [hunk 55]
```diff
@@ -0,0 +1,304 @@
...
+      id: 'actions',
+      header: 'Actions',
+      cell: (info) => {
+        return (
+          <button
+            className={classNames(
+              'text-[14px] text-white font-bold py-[5px] px-[20px] rounded-full',
+              {
+                'bg-red-600': info?.row?.original?.banned == false,
+                'bg-green-600': info?.row?.original?.banned == true,
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/Users.tsx]:
```suggestion
                'bg-red-600': !info?.row?.original?.banned,
                'bg-green-600': info?.row?.original?.banned,
```

File: apps/admin/src/pages/Users/Users.tsx [hunk 56]
```diff
@@ -0,0 +1,304 @@
...
+  });
+
+  return (
+    <div className='w-full h-full bg-white p-[30px] flex flex-col gap-[20px]'>
+      <p className='text-gray-900 text-3xl font-bold'>Users</p>
+      <input
+        type='text'
+        placeholder='Search'
+        value={search}
+        className='w-full h-[50px] rounded-[10px] border-[1px] border-gray-300 p-[10px] focus:outline-none focus:border-[#3f51b5]'
```
Comment (ivanms1) [inline apps/admin/src/pages/Users/Users.tsx]: add colors to tailwind config or use a default one

**Review by ivanms1 — CHANGES_REQUESTED** [review]

File: apps/admin/next.config.js (hunk 36 above)
Comment (revengemiroz) [reply apps/admin/next.config.js]: done

File: apps/admin/src/components/BarGraph/index.tsx (hunk 37 above)
Comment (revengemiroz) [reply apps/admin/src/components/BarGraph/index.tsx]: done

File: apps/admin/src/pages/Users/Users.tsx (hunk 55 above)
Comment (revengemiroz) [reply apps/admin/src/pages/Users/Users.tsx]: done

File: apps/admin/src/pages/Users/Users.tsx (hunk 54 above)
Comment (revengemiroz) [reply apps/admin/src/pages/Users/Users.tsx]: done

File: apps/admin/src/pages/Users/Users.tsx (hunk 56 above)
Comment (revengemiroz) [reply apps/admin/src/pages/Users/Users.tsx]: done

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**